src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -86,6 +95,19 @@ class FeatureSearchStore implements IFeatureSearchStore {
         const validatedSortOrder =
             sortOrder === 'asc' || sortOrder === 'desc' ? sortOrder : 'asc';
 
+        const featureLifecycleEnabled =
+            this.flagResolver.isEnabled('featureLifecycle');
+        const latestLifecycleStageQuery = this.db
```

Comment:
Also can we look into opportunities to pull it out as a method, and put all selects, joins and query there